skill-tree /src/main/java/com/RDS/skilltree/Endorsement/EndorsementController.java

```diff
@@ -0,0 +1,25 @@
+package com.RDS.skilltree.Endorsement;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("${API_V1_PREFIX}/endorsements")
```

Comment:
Nit: I feel the `API_V1_PREFIX` is not required though, we can directly write `/v1/path`

Reply:
I think having it a constant is good, We will not have to specify the path in every route

Reply:
okay

Reply:
But in other endpoints I don't think its being followed, we will have to change them too.